src/main/java/seedu/address/logic/commands/GetPrev3MonthFeeCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.fee.Month;
+import seedu.address.model.fee.Year;
+
+/**
+ * Gets current month and previous 2 months expected monthly fee based on the current list of sessions
+ */
+public class GetPrev3MonthFeeCommand extends Command {
+    public static final String COMMAND_WORD = "3Month_Fee";
```

Comment:
Might be consistent with the other commands if not capitalised

Reply:
Ahhh right! I have changed it to 3_month_fees instead.